### PR TITLE
feat(specs): add estimate path and responses [skip-bc]

### PR DESCRIPTION
**Pull request by cdhawke**
## 🧭 What and Why

🎟 JIRA Ticket: [OPTIM-1191]

### Changes included:

- Added a new path for `/estimate` in the abtests endpoints, as well as a new response object definition

[OPTIM-1191]: https://algolia.atlassian.net/browse/OPTIM-1191?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: specs/abtesting/paths/estimate.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+post:
```
[the remark this answers is not in the document]
Comment (cdhawke) [reply specs/abtesting/paths/estimate.yml:1]: Thanks I did this locally and forgot to push before I left for the day. I'll send it tomorrow 👍

**Review by febeck — APPROVED** [review]

**Comment by millotp** [comment]
Het @cdhawke, let me know if you need any help with the CI

**Comment by cdhawke** [comment]
@millotp thanks 😄 - I think i'm just working through getting the test added properly. If this next commit doesn't make some progress i'll reach out - I can't get the yarn cli commands to work locally so i'm just leveraging the CI to tell me what's wrong.

**Comment by millotp** [comment]
> I can't get the yarn cli commands to work locally

I would be interested to see which errors you are running into if you have time, this is something we could add to the contribution guides

**Comment by millotp** [comment]
> Do we just accept it as a breaking change to the API in this case

If this is what the API expect we can allow the breaking change yes

**Comment by cdhawke** [comment]
@millotp - feel free to merge this whenever if you're good with it. There's nothing blocking it

**Pull request merged**